src/test/DummyRemoteSimulator.ts

```diff
@@ -38,39 +38,32 @@ export class DummyRemoteSimulator extends RemoteSimulator {
         setInterval(this.broadcast.bind(this), 200);
     }
 
-    private getDataBundle(frameNumber: number, bundleSize: number): string {
+    private getDataBundle(frameNumber: number, bundleSize = 1): string {
         const msg: VisDataMessage = {
             msgType: NetMessageEnum.ID_VIS_DATA_ARRIVE,
             bundleStart: frameNumber,
             bundleSize: bundleSize,
             bundleData: [],
             fileName: this.lastRequestedFile,
         };
-
-        const bundleData: VisDataFrame[] = [];
-        for (let i = 0; i < bundleSize; i++) {
-            const data: VisDataFrame = {
-                frameNumber: frameNumber,
-                time: frameNumber * this.timeStep,
-                data: [
-                    1000,
-                    0,
-                    43,
-                    Math.cos(frameNumber / 4) * 5,
-                    Math.sin(frameNumber / 4) * 5,
-                    0,
-                    0,
-                    0,
-                    10,
-                    1,
-                    0,
-                ],
-            };
-            bundleData.push(data);
-            frameNumber++;
-        }
-
-        msg.bundleData = bundleData;
+        const data: VisDataFrame = {
```

Comment:
If I'm understanding right, the bundleSize would have only varied when the backend was sending more than one frame per message.

Reply:
I guess that's true! So are you not testing that scenario because the client can't handle it anymore, or because it just will never happen?  Because you might as well just get rid of the bundleSize arg to `getDataBundle` then